format/index.md

```diff
@@ -592,16 +602,6 @@ The following characters **MUST NOT** be used in member names:
 Data, including resources and relationships, can be fetched by sending a
 `GET` request to an endpoint.
 
-Clients **MUST** indicate that they can accept the JSON API media type, per
```

Comment:
Rather than saying that "Clients **MUST** indicate that they can accept the JSON API media type", I'm saying above that "Servers **MUST NOT** send responses with the `application/vnd.api+json` media type if the client has not indicated...it can understand this format." This seems to get at what we want—namely, you must ask for JSON API data in order to get it—while giving clients the freedom to ask for the other representations that, as our non-normative note alluded to, can exist at the same endpoint.